firefox-ios/Shared/Strings.swift

```diff
@@ -6935,13 +6930,13 @@ extension String {
     public static let TabTrayVisibleTabRangeAccessibilityHint = MZLocalizedString(
         key: "Tab %@ of %@",
         tableName: nil,
-        value: nil,
-        comment: "Message spoken by VoiceOver saying the position of the single currently visible tab in Tabs Tray, along with the total number of tabs. E.g. \"Tab 2 of 5\" says that tab 2 is visible (and is the only visible tab), out of 5 tabs total.")
+        value: "Tab %1$@ of %2$@",
+        comment: "Message spoken by VoiceOver saying the position of the single currently visible tab in Tabs Tray (%1$@), along with the total number of tabs (%2$@). E.g. “Tab 2 of 5” says that tab 2 is visible (and is the only visible tab), out of 5 tabs total.")
     public static let TabTrayVisiblePartialRangeAccessibilityHint = MZLocalizedString(
         key: "Tabs %@ to %@ of %@",
         tableName: nil,
-        value: nil,
-        comment: "Message spoken by VoiceOver saying the range of tabs that are currently visible in Tabs Tray, along with the total number of tabs. E.g. \"Tabs 8 to 10 of 15\" says tabs 8, 9 and 10 are visible, out of 15 tabs total.")
+        value: "Tabs %1$@ to %2$@ of %3$@",
```

Comment:
Example of value that diverged between .string and .swift. Interestingly enough, on the localization side we get the one in the .strings file.